printrun/projectlayer.py

```diff
@@ -581,29 +618,32 @@ def parse_3DLP_zip(self, name):
         return ol, -1, "bitmap"
 
     def load_file(self, event):
-        dlg = wx.FileDialog(self, ("Open file to print"), style = wx.FD_OPEN | wx.FD_FILE_MUST_EXIST)
-        dlg.SetWildcard(("Slic3r or Skeinforge svg files (;*.svg;*.SVG;);3DLP Zip (;*.3dlp.zip;)"))
+        dlg = wx.FileDialog(self, _("Open file to print"), style = wx.FD_OPEN | wx.FD_FILE_MUST_EXIST)
+        dlg.SetWildcard(_("Slic3r or Skeinforge svg files (;*.svg;*.SVG;);3DLP Zip (;*.3dlp.zip;)"))
         if dlg.ShowModal() == wx.ID_OK:
             name = dlg.GetPath()
             if not(os.path.exists(name)):
-                self.status.SetStatusText(("File not found!"))
+                self.status.SetStatusText(_("File not found!"))
                 return
             if name.endswith(".3dlp.zip"):
                 layers = self.parse_3DLP_zip(name)
                 layerHeight = float(self.thickness.GetValue())
             else:
                 layers = self.parse_svg(name)
-                layerHeight = layers[1]
-                self.thickness.SetValue(str(layers[1]))
-                print("Layer thickness detected:", layerHeight, "mm")
-            print(len(layers[0]), "layers found, total height", layerHeight * len(layers[0]), "mm")
+                layerHeight = float(f"{layers[1]:.2f}") # layer[1] formatted
+                # Caution: *1000000 is a special adjustment to dislpay the Slic3r 1.3.0 *.svg files correctly.
+                # layerHeight = float(f"{layers[1]*1000000:.2f}")
```

Comment:
Files created with Slic3r 1.3.0 have their z-height stored in a weird way. But I didn't like this hack here, because it messes with older files (like the examples in the testfile folder).

Reply:
What are we really doing here? Isn't `layers[1]` already a float? Why convert it to a string and then back a to a float? Wouldn't it be cleaner to do `layerHeight = round((layers[1], 2)`?